views/scss/test-runner.scss

```diff
@@ -50,14 +50,41 @@
         &.horizontal-action-bar {
             opacity: 0;
 
-            .progress-box, .timer-box, .title-box {
-                padding-top: 4px
+            .title-box {
+                padding-top: 4px;
+                overflow: hidden;
+                white-space: nowrap;
+                max-width: 100%;
+            }
+            .progress-box, .timer-box, .item-number-box {
+                padding-top: 4px;
+                display: inline-block;
+                vertical-align: top;
+                float: right;
             }
             .progressbar {
                 margin-top: 5px;
                 min-width: 150px;
                 height: 0.6em;
             }
+            &.top-action-bar > .control-box {
+                position: relative;
+                & .control-box-group {
+                    position: absolute;
+                    top: 0;
+                    right: -5px;
+                    padding-right: 5px;
+
+                    background: -moz-linear-gradient(left,  transparent 0%, whiten($info, .1) 30px);
```

Comment:
Also mixin `linear-gradient` use `background: linear-gradient( #{$css} );` for setting gradient, that not works in Chrome, because Chrome require -webkit prefix. I guess it should be solved by autoprefixer, but it isn't in dependencies.